src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx

```diff
@@ -11,7 +11,7 @@ redirects:
   - /docs/telemetry-data-platform/get-started/manage-data/manage-stored-data#adjust-retention
 ---
 
-Review and manage how long your data will be stored in the New Relic database (NRDB) on the **Data retention** page. This page helps you understand the sources of data that you’re storing, when they were last updated, and the length the data is stored. Retention settings range from 8 to 395 days depending on the data's `source` and `namespace` classifications in New Relic. 
+Using the **Data retention** page, review and manage how long your data will be stored in the New Relic database (NRDB). Keep data for shorter periods to protect sensitive information or for longer periods to notice trends over time. At a glance, get a the sources of data that you’re storing, when they were last updated, and the length the data is stored. Retention settings range from 8 to 395 days depending on the data's `source` and `namespace` classifications in New Relic. 
```

Comment:
"At a glance, get a the sources" -- is something missing between "a" and "the" or should "a" simply be deleted?